src/masternode-sync.cpp

```diff
@@ -508,6 +508,21 @@ void CMasternodeSync::ProcessTick()
     ReleaseNodes(vNodesCopy);
 }
 
+void CMasternodeSync::SendGovernanceSyncRequest(CNode* pnode, const char* strMessage, uint256 nHash)
```

Comment:
I thought it was used on line 348, but there the parameter is actually the default as well, I guess you're right.